charts/asset-manager/templates/deployment.yaml

```diff
@@ -94,3 +97,8 @@ spec:
       - name: {{ .Release.Name }}-nginx-conf
         configMap:
           name: {{ .Release.Name }}-nginx-conf
+      - name: {{ .Release.Name }}-asset-uploads-efs
+        nfs: 
+          server: "{{ required "An NFS server is required!" .Values.nfsServer }}"
+          path: /
+ 
```

Comment:
tiny nit: something funny going on with the line endings here (looks like a blank line followed by a missing line ending at EOF, somehow - check your editor config?)

Reply:
👍